examples/incorrect/arithemtic_operators_syntax_errors.txt

```diff
@@ -0,0 +1,13 @@
+# some syntax errors
++ 4;
```

Comment:
I do not think unary plus should cause an error.

Reply:
for now we can leave it like this, let's start with binary operators only and then introduce unary operators